### PR TITLE
Add conda-forge recipe for calysto_bash

### PR DESCRIPTION
**Pull request by BoPeng**
Checklist

- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x ] Source is from official source
- [ x] Package does not vend other packages
- [x ] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/calysto_bash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - pip
+    - python
+    - ipykernel
+  run:
+    - python
+    - ipykernel
+    - jupyter_client
+    - metakernel
+    - bash  # [unix]
+    - m2_bash  # [win]
```
Comment (hmaarrfk) [inline recipes/calysto_bash/meta.yaml]:
```suggestion
    - m2-bash  # [win]
```

File: recipes/calysto_bash/meta.yaml [hunk 2]
```diff
@@ -20,6 +20,8 @@ requirements:
     - pip
     - python
     - ipykernel
+  build:
+    - metakernel
```
Comment (hmaarrfk) [inline recipes/calysto_bash/meta.yaml]: Shouldn't this be a host dependency?

Comment (BoPeng) [reply recipes/calysto_bash/meta.yaml]: Sorry, I do not really know the difference between `host` and `build`. I see that  `python -m calysto_bash install` failed due to lack of `metakernel` so I added it, assuming it is required for installation (build).

Comment (hmaarrfk) [reply recipes/calysto_bash/meta.yaml]: i think it is mostly compilers go in build. Not too clear on that either.

**Comment by BoPeng** [comment]
@blink1073 The license fails due to lack of LICENSE file. Please accept https://github.com/Calysto/calysto_bash/pull/3 (or add another license) and make a new pipy release.

**Comment by hmaarrfk** [comment]
You can add the license file manually to the recipe.

File: recipes/calysto_bash/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  imports:
+    - calysto_bash
+  commands:
+    - jupyter kernelspec list | grep bash
+    - jupyter run --kernel=bash command.sh
+  files:
+    - command.sh
+
+about:
+  home: https://github.com/takluyver/bash_kernel
```
Comment (hmaarrfk) [inline recipes/calysto_bash/meta.yaml]: this doesn't seem right either.

**Comment by BoPeng** [comment]
But I do not know what license @blink1073 wants to use for `calysto_bash`. I think he can disregard the PR and add LICENSE directly to this recipe.

**Comment by hmaarrfk** [comment]
thats fair.

File: recipes/calysto_bash/meta.yaml (hunk 3 above)
Comment (BoPeng) [reply recipes/calysto_bash/meta.yaml]: Fixed, things like that happens when there is a baby crawling around you. :-)

File: recipes/calysto_bash/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - metakernel
+    - bash  # [unix]
+    - m2-bash  # [win]
+
+test:
+  imports:
+    - calysto_bash
+  commands:
+    - jupyter kernelspec list | grep bash
+    - jupyter run --kernel=bash command.sh
```
Comment (blink1073) [inline recipes/calysto_bash/meta.yaml]: The kernel is called `calysto_bash`: https://github.com/Calysto/calysto_bash/blob/dfa6833187e1fe9d9c229fd7bcc839fd7813d74b/calysto_bash/kernel.json#L8

Comment (BoPeng) [reply recipes/calysto_bash/meta.yaml]: Sorry. Fixed.

**Comment by BoPeng** [comment]
@hmaarrfk Not sure why the appveyor test fail because the azure windows test passed without problem.

File: recipes/calysto_bash/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/calysto_bash/meta.yaml]: Are these even needed in host.  They seem to be runtime deps.  host deps are those that are required for install on the target machine architecture

**Comment by hmaarrfk** [comment]
I think it is a little beyond me. @mariusvniekerk has said that sometimes there are glitches in the windows builds, but so long as one of them passes, he was OK with it..... not too sure.

File: recipes/calysto_bash/meta.yaml (hunk 2 above)
Comment (BoPeng) [reply recipes/calysto_bash/meta.yaml]: The problem here is that the `install` step (`python -m calysto_bash install`) is handled by `metakernel`.

**Comment by BoPeng** [comment]
I do not know any way to debug the AppVeyor build. Perhaps we can merge this PR and test if it works all right under windows?

**Comment by scopatz** [comment]
This is fine. Thanks!

**Comment by BoPeng** [comment]
The windows binary is not built https://anaconda.org/conda-forge/calysto_bash/files, so I suppose we will have to fix the AppVeyor test.

**Pull request merged**